lib/drivers/roche/accuChekSmartPix.js

```diff
@@ -0,0 +1,616 @@
+import _ from 'lodash';
+import sundial from 'sundial';
+import xml from 'xml2js';
+import { get, set } from 'idb-keyval';
```

Comment:
It's so that the permission to the device persists across restarts of the app. IIRC I first tried using our `localStore` but for some reason that didn't work.

Reply:
Ah, I imagine that it's because `localStore` can only store strings where indexDB can store anything that can be passed through [structured clone](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm)

Reply:
Oh yeah, that was it! :+1: